pdc/apps/auth/views.py

```diff
@@ -903,3 +911,204 @@ def destroy(self, request, *args, **kwargs):
         On success, HTTP status code is 204 and the response has no content.
         """
         return super(GroupResourcePermissionViewSet, self).destroy(request, *args, **kwargs)
+
+
+class APIResourcePermissionViewSet(StrictQueryParamMixin,
+                                   ChangeSetCreateModelMixin,
+                                   ChangeSetDestroyModelMixin,
+                                   mixins.RetrieveModelMixin,
+                                   mixins.ListModelMixin,
+                                   MultiLookupFieldMixin,
+                                   viewsets.GenericViewSet):
+    """
+    This end-point provides API resource permissions.
+    """
+    queryset = models.GroupResourcePermission.objects.all()
+    serializer_class = serializers.GroupResourcePermissionSerializer
```

Comment:
Should it use a new serializer and implement your logic in that serializer? 

Should this API support create method? It looks duplicate of group resource permission API.